variables.tf

```diff
@@ -84,7 +84,7 @@ variable "bucket_enable_logs" {
 
 variable "bucket_enable_mfa_delete" {
   type        = bool
-  default     = true
+  default     = false
```

Comment:
uhmm, changing defaults will cause everyone using this module to apply changes, do we want that?

Lets chat about this change over zoom.

Reply:
Talked with Alan about this, we need to set this to false by default since MFA delete requires MFA Authentication enabled. If it is not, users will have this error:
```
Error: Error putting S3 versioning: AccessDenied: Mfa Authentication must be used for this request
	status code: 403, request id: 123ac, host id: 123abc=
```